[Doc] Add FAQ for integration on the lower minimum deployment target (#235)
Azure/communication-ui-library-ios: docs/manual-installation.md

    1. Update your target scheme with `ENABLE_ADDRESS_SANITIZER = YES`. For more instruction, please visit [How do you enable Clang Address Sanitizer in Xcode?](https://stackoverflow.com/questions/32150924/how-do-you-enable-clang-address-sanitizer-in-xcode). 
    
### FAQ
#### How to conditionally use the UI library based on the iOS version when the project has a lower minimum deployment target?

> The UI library wouldn't work for the device target lower than iOS 14. But it's still possible to import the UI library with the following workaround and conditional launch the UI library when iOS 14 is available. 

Reviewer: ```suggestion
The UI library won't work for devices lower than iOS 14 but it's still possible to import the UI library with the following workaround and then conditionally launch the UI library when iOS 14 is available. 
```

1. Add a framework to your project’s target and set the minimum deployment target > iOS 14. 

> 	* Use this framework for all needed dependencies import, through Cocoapods or manual embed.

Reviewer: ```suggestion
	* Use this framework for all needed dependencies imported through Cocoapods or manual embed.
```

	* Create an UIViewController only targeted for the framework.
	* Import needed dependency modules inside the UIViewController, add `tokenCredential`, `callCompositeOptions`, and code in the `init()` method to launch the UI library. 
	* Set this UIViewController as the `NSPrincipalClass` for the newly added framework. 
	* If needed, the principal class could play a complex role like returning different classes by implementing the protocol from the main target.
2. Embed the newly added framework into your main target's "Frameworks and Libraries" section with `Embed and Sign`.

> 3. In main app, use the [`Loadable Bundles in Cocoa`](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/LoadingCode/Concepts/CocoaBundles.html#//apple_ref/doc/uid/20001269-BAJCIAHA) to load the new added UIViewController from the framework when the iOS 14 is available. Code example:

Reviewer: ```suggestion
3. In main app, use the [`Loadable Bundles in Cocoa`](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/LoadingCode/Concepts/CocoaBundles.html#//apple_ref/doc/uid/20001269-BAJCIAHA) to load the newly added UIViewController from the framework when the iOS 14 is available. Code example:
```

```
if #available(iOS 14.0, *) {
    if let principleClass = Bundle(identifier: "com.****.UILibraryFramework")?.principalClass as? UIViewController.Type {
        let controller = principleClass()
        self.addChild(controller)
        view.addSubview(controller.view)
    }
}
```

> 

Reviewer 2: Remove these extra lines?
